site/siteConfig.jsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import siteConfig from 'terra-site/lib/config/site.config';
```

Comment:
The [default `siteConfig`](https://github.com/cerner/terra-site/blob/dynamic-site/src/config/site.config.jsx) provides the default navigation for the component rendering with path as `/site/components` and the tests rendering with path as `/tests`. The order of navigation matters for NavTab organization.

Additionally, one can configure the `appLogoSrc` to provide an img src for the [`placeholder background`](https://github.com/cerner/terra-site/blob/dynamic-site/src/app/common/Placeholder.jsx#L9). This placeholder is rendered when a route is hit without a specific component rendering.